### ISSUE TITLE
Crash on launch in Kodi 19

### ISSUE THREAD

**Issue by luciusbono** [plexinc/plex-for-kodi#335]
## Description:
The Plex for Kodi addon (installed from the repository built into Kodi 19.0) doesn't launch on my machine. Happy to provide any other logs, but tailing `~/.kodi/temp/kodi.log` gives me the following:

```
2020-11-01 21:32:30.055 T:11631   ERROR <general>: Control 50 in window 10001 has been asked to focus, but it can't
2020-11-01 21:32:30.103 T:12215    INFO <general>: initializing python engine.
2020-11-01 21:32:30.323 T:12215 WARNING <general>: xbmc.translatePath is deprecated and might be removed in future kodi versions. Please use xbmcvfs.translatePath instead.
2020-11-01 21:32:30.324 T:12215   ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'AttributeError'>
                                                   Error Contents: module 'kodi_six.xbmc' has no attribute 'LOGNOTICE'
                                                   Traceback (most recent call last):
                                                     File "/home/[redacted/.kodi/addons/script.plex/plugin.py", line 7, in <module>
                                                       from lib import _included_packages, plex, util
                                                     File "/home/[redacted]/.kodi/addons/script.plex/lib/plex.py", line 14, in <module>
                                                       from . import util
                                                     File "/home/[redacted]/.kodi/addons/script.plex/lib/util.py", line 59, in <module>
                                                       def LOG(msg, level=xbmc.LOGNOTICE):
                                                   AttributeError: module 'kodi_six.xbmc' has no attribute 'LOGNOTICE'
                                                   -->End of Python script error report<--
                                                   
2020-11-01 21:32:30.335 T:11646    INFO <general>: CActiveAESink::OpenSink - initialize sink
2020-11-01 21:32:30.356 T:12215    INFO <general>: Python interpreter stopped
2020-11-01 21:32:30.365 T:12214   ERROR <general>: GetDirectory - Error getting plugin://script.plex/
2020-11-01 21:32:30.367 T:11631   ERROR <general>: CGUIMediaWindow::GetDirectory(plugin://script.plex/) failed
2020-11-01 21:32:30.386 T:11646    INFO <general>: PulseAudio: Opened device Default in pcm mode with Buffersize 150 ms Periodsize 50 ms
2020-11-01 21:32:35.606 T:11646    INFO <general>: CActiveAESink::OpenSink - initialize sink
2020-11-01 21:32:35.612 T:11646    INFO <general>: PulseAudio: Opened device Default in pcm mode with Buffersize 150 ms Periodsize 50 ms
```

It looks like the repo is pulling: [develop-kodi19](https://github.com/plexinc/plex-for-kodi/tree/develop-kodi19) (just going by the version number). I recognize this is all alpha but I wanted to raise this issue since this is now the "golden path" for a lot of Fedora 33 users (F33 _just_ came out). 

## Details:
- Add-on version: 0.3.1
- Kodi version: 19.0 (19.0.0.5.20201005git54be31b.fc33)
- OS Platform: Fedora 33
- Hardware: X86 

## Checklist
- [x] I have included a link to a log (or at the very least a link to a forum post with a log) from a session that had the issue
- [ ] I have added appropriate GitHub labels

**Comment by pannal** [comment]
Please check whether [my develop branches](https://github.com/pannal/plex-for-kodi) fix your issue.

**Comment by MrPumo** [comment]
Proposed PR #328

**Comment by pannal** [comment]
Yes I incorporated that into my develop branches.

**Comment by razihaider-694** [comment]
This problem is very common you can follow the below fixes-
Uninstall the addon and reinstalled it.
Use VPNs.
Restart your device.

I hope it will work in further if you will face the addon not working problem then you can go for the https://thefiresticktv.com/sportsdevil-kodi-addon/

**Comment by ruuk** [comment]
Fixed in #328
